src/components/BarChart/BarChart.jsx

```diff
@@ -163,11 +160,22 @@ const BarChart = createClass({
 		yAxisTitleColor: number,
 	},
 
+	statics: {
+		PADDING: 0.05,
+		PADDING_GROUPED_OR_STACKED: 0.3,
+		MARGIN: {
+			top: 10,
+			right: 20,
+			bottom: 50,
+			left: 80,
+		},
+	},
+
 	getDefaultProps() {
 		return {
 			height: 400,
 			width: 1000,
-			margin: {
+			margin: { // duplicated because `statics` aren't available during getDefaultProps
```

Comment:
I tried that, but it makes it unclear what the defaults are when looking at the docs. It also isn't clear that they are available as statics. Tradeoffs :/ We should probably land on a pattern for this across our components.